20151222/rails-5.0.0.beta1-demo/test/controllers/comments_controller_test.rb

```diff
@@ -0,0 +1,53 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.